src/Learning/MHHistograms/MHDiscretizerMODLHistogram.cpp

```diff
@@ -109,33 +111,50 @@ void MHDiscretizerMODLHistogram::DiscretizeValues(ContinuousVector* cvSourceValu
 			oaResultHistograms.Add(postprocessedOptimizedHistogram);
 		oaResultHistograms.Add(optimizedHistogram);
 
-		// Recherche du meilleur histogramme
-		if (postprocessedOptimizedHistogram != NULL)
-			bestHistogram = postprocessedOptimizedHistogram;
-		else
-			bestHistogram = optimizedHistogram;
-
-		// Cas avec contrainte sur le nombre max d'intervalles
+		// Les histogrammes MODL sont optimaux.
+		// Pour chaque variable numerique, on produit une serie d'histogrammes interpretables, par granularite croissante.
+		// Le probleme est que l'histogramme interpretable le plus fin est parfois tres complexe, de type "herisson",
+		// ce qui peut derouter un utilisateur non expert. Ce probleme se produit dans environ 20% des cas.
+		// Plutot que de montrer par defaut l'histogramme interpretable le plus fin, on propose d'utiliser la
+		// regle de Terell-Scott  (cf. https://en.wikipedia.org/wiki/Histogram), avec un nombre de bins de (2 N)^1/3
```

Comment:
Typo: "Terrell-Scott" (add "r").
Wikipedia reference: perhaps a more specific one is https://en.wikipedia.org/wiki/Scott%27s_rule#Terrell%E2%80%93Scott_rule ?

Reply:
Fait partout, dans le code et les commentaires

Reply:
I can see the typo fix, but not the Wikipedia page reference update...

Reply:
Je préfère laisser la ref plus génériques aux histogrammes (après Terrell-Scott est facile à retrouver)

Reply:
Ok, then how about this compromise: "https://en.wikipedia.org/wiki/Histogram#Terrell%E2%80%93Scott_rule" ?